### PR TITLE
state executor: groom logs

### PR DESCRIPTION
**Pull request by alexanderbez**
ref: #5912

### CODE REVIEW

File: state/execution.go [hunk 1]
```diff
@@ -234,10 +234,10 @@ func (blockExec *BlockExecutor) Commit(
 	// ResponseCommit has no error code - just data
 
 	blockExec.logger.Info(
-		"Committed state",
+		"committed state",
 		"height", block.Height,
-		"txs", len(block.Txs),
-		"appHash", res.Data,
+		"num_txs", len(block.Txs),
+		"app_hash", fmt.Sprintf("%X", res.Data),
```
Comment (alexanderbez) [inline state/execution.go:240]: @melekes I see PR https://github.com/tendermint/tendermint/pull/5960 which states that this should be logged as hex without the need to format, right? Is that feature currently live in Stargate?

Comment (melekes) [reply state/execution.go:240]: > Is that feature currently live in Stargate?

I think so.

> ��\x1c1�U��ȺXu}���D�\x1c\v߃#OY���8\x13\x19.

weird...

Comment (alexanderbez) [reply state/execution.go:240]: This works regardless of the underlying logger, right (i.e. go-kit vs zerolog, etc...)? In other words, we apply this logic before the underlying logger API gets called, correct?

Comment (melekes) [reply state/execution.go:240]: > This works regardless of the underlying logger, right (i.e. go-kit vs zerolog, etc...)? In other words, we apply this logic before the underlying logger API gets called, correct?

correct. here is the code: https://github.com/tendermint/tendermint/blob/93719c2eb34670b7ae541384ec1b86c08fb327de/libs/log/tmfmt_logger.go#L86-L89

Comment (melekes) [reply state/execution.go:240]: maybe it's because AppHash has a type `tmbytes.HexBytes`, not `[]byte`

Comment (alexanderbez) [reply state/execution.go:240]: Ahh, that explains it then. The SDK does not use `NewTMFmtLogger`. It uses a zerolog logger directly. I think we should keep this change then.

**Review by cmwaters — COMMENTED** [review]
Are you going to decapatlilize the logs? There are still a few in execution.go that are capatilized

**Comment by alexanderbez** [comment]
I'll address the rest in `execution.go`, thanks @cmwaters. Once I do, we can merge and backport.

**Review by cmwaters — APPROVED** [review]
👍

**Pull request merged**